### ISSUE TITLE
errorMessage: "Could not authenticate with Plex"

### ISSUE THREAD

**Issue by JohnLindahlTech** [Ombi-app/Ombi#4142]
**Describe the bug**
When I try to log in with an approved Plex user Ombi just spins.
Network tab gives the following message for request to path `/api/v1/token/[random token]`:
```json
{"errorMessage": "Could not authenticate with Plex"}
```

Note: I was successful in connecting to my plex instance with adminuser and importing both libraries and users.

**To Reproduce**
1. Go to `/login` 
2. Click on `log in with Plex`
3. Log in with plex (successfully according to the popup)
4. Nothings happens visually, but see network tab problem above.

**Expected behavior**
Be able to log in

**Screenshots**
N/A

**Logs (Logs directory where Ombi is located)**
I can see no relevant logs when loading `/Settings/Logs` -> `"today"`, is there a better way to look (with the docker image)?

**Desktop (please complete the following information):**
 - OS: Windows 10 and/or iOS 14

**Ombi Version (please complete the following information):**
 - Version 4.0.1328.0 (Docker image:`linuxserver/ombi:development-version-v4.0.1328`)
- Media Server: Plex (v1.21.4.4079)
- Database Type: SQLite

**Additional context**
N/A

**Comment by tidusjar** [comment]
Is the user you are trying to auth with in Ombi? Did you use the user importer to get that user in?

**Comment by JohnPhoto** [comment]
Yes, I am trying to log in with the user that is admin of my plex-server, it is also the account that I synced everything from Plex with.

Also: It did not help to delete the plex-user from ombi and re-sync, the accound came back, but still not working.

**Comment by tidusjar** [comment]
Logs are in the folder that you set to the config variable. Can you try to login the. Upload a log?

**Comment by JohnPhoto** [comment]
Sure thing from `[]/Logs`

**Comment by JohnPhoto** [comment]
Sure thing! Not giving much though:
From `[/config-mount]/Logs/log-20210410.txt`, (I logged in at `2021-04-10 16:01:04.xxx +02:00`):
```
2021-04-10 16:01:01.315 +02:00 [Information] End processing HTTP request after 81.6159ms - 200

[Started loggin in with plex, and approved and according to Plex popup, successful, but just spinning in Ombi from here]

2021-04-10 16:01:05.024 +02:00 [Information] Start processing HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:05.025 +02:00 [Information] Sending HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:05.086 +02:00 [Information] Received HTTP response headers after 61.5594ms - 200
2021-04-10 16:01:05.086 +02:00 [Information] End processing HTTP request after 61.7584ms - 200
2021-04-10 16:01:09.071 +02:00 [Information] Start processing HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:09.071 +02:00 [Information] Sending HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:09.135 +02:00 [Information] Received HTTP response headers after 64.4111ms - 200
2021-04-10 16:01:09.135 +02:00 [Information] End processing HTTP request after 64.5886ms - 200
2021-04-10 16:01:11.853 +02:00 [Information] Start processing HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:11.853 +02:00 [Information] Sending HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:11.942 +02:00 [Information] Received HTTP response headers after 88.3737ms - 200
2021-04-10 16:01:11.942 +02:00 [Information] End processing HTTP request after 88.5727ms - 200
2021-04-10 16:01:15.859 +02:00 [Information] Start processing HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:15.859 +02:00 [Information] Sending HTTP request "GET" https://plex.tv/api/v2/pins/159756791
2021-04-10 16:01:15.980 +02:00 [Information] Received HTTP response headers after 121.1828ms - 200
2021-04-10 16:01:15.980 +02:00 [Information] End processing HTTP request after 121.3628ms - 200
2021-04-10 16:01:16.833 +02:00 [Information] Start processing HTTP request "GET" https://webservice.fanart.tv/v3/movies/399106
2021-04-10 16:01:16.833 +02:00 [Information] Sending HTTP request "GET" https://webservice.fanart.tv/v3/movies/399106
2021-04-10 16:01:17.545 +02:00 [Information] Received HTTP response headers after 712.2197ms - 200
2021-04-10 16:01:17.545 +02:00 [Information] End processing HTTP request after 712.4233ms - 200
2021-04-10 16:01:17.587 +02:00 [Information] Start processing HTTP request "GET" https://webservice.fanart.tv/v3/tv/280619
2021-04-10 16:01:17.587 +02:00 [Information] Sending HTTP request "GET" https://webservice.fanart.tv/v3/tv/280619
2021-04-10 16:01:17.653 +02:00 [Information] Received HTTP response headers after 65.7122ms - 200
2021-04-10 16:01:17.653 +02:00 [Information] End processing HTTP request after 65.9338ms - 200
2021-04-10 16:01:19.865 +02:00 [Debug] Batch acquisition of 0 triggers
2021-04-10 16:01:21.590 +02:00 [Information] Start processing HTTP request "GET" https://webservice.fanart.tv/v3/movies/278
2021-04-10 16:01:21.590 +02:00 [Information] Sending HTTP request "GET" https://webservice.fanart.tv/v3/movies/278
2021-04-10 16:01:21.656 +02:00 [Information] Received HTTP response headers after 65.9055ms - 200
2021-04-10 16:01:21.656 +02:00 [Information] End processing HTTP request after 66.1181ms - 200
2021-04-10 16:01:21.657 +02:00 [Information] Start processing HTTP request "GET" https://webservice.fanart.tv/v3/tv/81189
2021-04-10 16:01:21.657 +02:00 [Information] Sending HTTP request "GET" https://webservice.fanart.tv/v3/tv/81189
2021-04-10 16:01:21.691 +02:00 [Information] Received HTTP response headers after 33.9837ms - 200
2021-04-10 16:01:21.691 +02:00 [Information] End processing HTTP request after 34.1799ms - 200
2021-04-10 16:01:25.651 +02:00 [Information] Start processing HTTP request "GET" https://plex.tv/api/v2/pins/2088694107
2021-04-10 16:01:25.651 +02:00 [Information] Sending HTTP request "GET" https://plex.tv/api/v2/pins/2088694107
2021-04-10 16:01:25.850 +02:00 [Information] Received HTTP response headers after 198.7702ms - 200
2021-04-10 16:01:25.850 +02:00 [Information] End processing HTTP request after 198.955ms - 200
2021-04-10 16:01:29.646 +02:00 [Information] Start processing HTTP request "GET" https://plex.tv/api/v2/pins/2088694107
2021-04-10 16:01:29.646 +02:00 [Information] Sending HTTP request "GET" https://plex.tv/api/v2/pins/2088694107
2021-04-10 16:01:29.713 +02:00 [Information] Received HTTP response headers after 66.7188ms - 200
2021-04-10 16:01:29.713 +02:00 [Information] End processing HTTP request after 66.9292ms - 200

```

Also tested with other synced plex-accounts, same thing.

**Comment by woogerboy21** [comment]
Just wanted to throw in that I also am experiencing the same issue (running the linux-x64 version not docker), but cant say when the problem began.  My ombi is used so infrequently that I have completly forgot the local user account information.  Given I havent updated my ombi in some time it might be a change in the way the plex authentication mechanism works. If there is a way to tell what version of ombi is running without logging in I could tell you want version I have to give an idea on any development change timeframes.

**Comment by twanariens** [comment]
Should be fixed in the latest release V4.15.2
